src/test/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListUpdaterTest.java

```diff
@@ -119,6 +119,26 @@ void handlesCaseNumberForCpp() throws IOException {
         assertThat(dailyList.getStatus()).isEqualTo(PROCESSED);
     }
 
+    @Test
+    void handlesNoTimeMarkingNote() throws IOException {
+
+        var dailyListUser = new UserAccountEntity();
+        OffsetDateTime testTime = DateConverterUtil.toOffsetDateTime(HEARING_DATE);
+        when(currentTimeHelper.currentOffsetDateTime()).thenReturn(testTime);
+        when(systemUserHelper.getDailyListProcessorUser()).thenReturn(dailyListUser);
+        when(courthouseRepository.findByCourthouseName("SWANSEA")).thenReturn(Optional.of(new CourthouseEntity()));
+        HearingEntity hearing = new HearingEntity();
+        CourtCaseEntity courtCase = new CourtCaseEntity();
+        hearing.setCourtCase(courtCase);
+        LocalDateTime expectedHearingDate = LocalDateTime.of(2023, 9, 23, 9, 0, 0);
+        when(retrieveCoreObjectService.retrieveOrCreateHearing("SWANSEA", "1A", "42GD2391421", expectedHearingDate, dailyListUser))
+            .thenReturn(hearing);
+        DailyListEntity dailyList = setUpDailyList("handlesNoTimeMarkingNote.json");
+        dailyListUpdater.processDailyList(dailyList);
+
+        verify(hearingRepository, times(1)).saveAndFlush(hearingEntityCaptor.capture());
```

Comment:
We should validate the content of the capture

Reply:
In that case we can remove this verify and replace it with a verify for the mocked call on 134 (with the expected args)